ArduCopter/Parameters.h

```diff
@@ -256,7 +256,7 @@ class Parameters {
         //
         // 160: Navigation parameters
         //
-        k_param_rtl_altitude = 160,
+        k_param_rtl_altitude_old = 160, // deprecated 13/SEP/2021
```

Comment:
I wasn't 100% sure on how we are marking param conversions like this one?

```suggestion
        k_param_rtl_altitude_old = 160, // deprecated 13/SEP/2021
```

Reply:
no longer necessary when using convert_parameter_width